nekoyume/Assets/_Scripts/UI/Widget/Popup/OutfitInfoListPopup.cs

```diff
@@ -12,6 +13,15 @@ public class OutfitInfoListPopup : PopupWidget
         [SerializeField]
         private RandomOutfitScroll scroll;
 
+        [SerializeField]
+        private Button closeButton;
+
+        protected override void Awake()
+        {
+            base.Awake();
+            closeButton.onClick.AddListener(() => Close());
```

Comment:
```suggestion
            closeButton.onClick.AddListener(() => Close(true));
```
이거 애니메이션이 제대로 들어가있는지 모르겠어서 그냥 Close(true) 하는게 나을거같습니다

Reply:
닫을때 쪼그라드는건 제대로 나오는 것 처럼 보이는데 true로 해두고 테스트 함 해보겠습니다

Reply:
아 아니다 하지말죠